crud/common/schema.lua

```diff
@@ -80,18 +84,18 @@ end
 -- This wrapper is used for functions that can fail if router uses outdated
 -- space schema. In case of such errors these functions returns `need_reload`
 -- for schema-dependent errors.
-function schema.wrap_func_reload(func, ...)
+function schema.wrap_func_reload(vshard_router, func, ...)
```

Comment:
You are right, I've thought so too, but it is where it all became a little tricky.

After introducing sharding info reload, most crud internal methods are called with several wrappers (actually, two: space schema reload wrapper and sharding info reload wrapper). And it would be nice to have wrappers that use the same api. 

How it looked before the patch?
```lua
schema_wrapper(sharding_wrapper, func, ...)
```
which unwraps to into
```lua
function schema_wrapper(func, ...)
    local res, err = func(...)
end

function sharding_wrapper(func, ...)
    local res, err = func(...)
end
```

It was cool and it worked nice since none of the wrappers actually was interested in what is stored in `...`.

Now both wrappers need to know what `vshard_router` is used. But if I use
```lua
function schema_wrapper(func, vshard_router, ...)
    local res, err = func(...)
end

function sharding_wrapper(func, vshard_router, ...)
    local res, err = func(...)
end
```
it won't work with
```lua
schema_wrapper(sharding_wrapper, func, vshard_router, ...)
```
because schema wrapper second argument will be `func`, not a `vshard_router`.